mpmp/src/model/Plot.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package model;
+
+import java.util.HashSet;
+
+/**
+ *
+ * @author leander.dreier
+ */
+/**
+	 * A plot that can be bought, sold, turned into a hypothec, ...
+	 */
```

Comment:
It auto-generated a JavaDoc comment even though we have one. Also, the whole class is indented one tab too far.